### PR TITLE
#219 remove unused api routes

### LINKED ISSUE #219: Remove unused API routes
As a developer I would like to only have used **or** possibly interesting routes in the code to avoid messy left-overs.

---

### :thought_balloon: &nbsp; Hints & Thoughts
Please be very careful here and show some 'Weitsichtigkeit'

---

## :white_check_mark: &nbsp; Conditions of satisfaction

- [ ] Should remove unused routes
- [ ] Should check which unused routes could be interesting for use from the outside or could be needed in the future and therefore be left in the code. 
- [ ] Please include a short summary for each unused route that is kept, so that the reviewers can understand your thinking about keeping those routes

### PR DESCRIPTION
**Pull request by WolfgangDaniel**
closes #219 

## Changes
Over time a lot of unused api routes with tangling unused functionality accumulated. With this user story we delete all the unused routes and its functionality if the route is not expected to be used in the near future. 

## Checklist before merge

**Developer's responsibilities**
* [ ] **Assign** one or two developers
* [ ] **Change code** if reviewer(s) has/have requested it
* [ ] **Pull request build** has passed
* [ ] **tested locally** (in at least chrome & firefox if frontend)
* [ ] updated the **documentation**
* [ ] added **tests** where necessary

### CODE REVIEW

File: server/controllers/ssotRetrievalController.js
Comment (kej-jay) [inline server/controllers/ssotRetrievalController.js:167]: Please try if it works with dot notation or disable listing here

File: server/routes/ssot.js
Comment (kej-jay) [inline server/routes/ssot.js:21]: I guess the comment is good to know, but I am not sure if we should have it in here, as all the other comments should be gone now too. I'd leave this up to you

**Review by kej-jay — COMMENTED** [review]
Two very minor things, which I would like to be addressed before merging. Please notify me upon doing so.
Great job!

File: server/controllers/ssotRetrievalController.js
Comment (WolfgangDaniel) [reply server/controllers/ssotRetrievalController.js:167]: ✅ Is now in dot notation

File: server/routes/ssot.js
Comment (WolfgangDaniel) [reply server/routes/ssot.js]: I forgot to delete it. I have deleted it now ✅

**Review by kej-jay — APPROVED** [review]
Looks good to me now. Thank you!

**Review by lukashueller — APPROVED** [review]
Looks good for me.

**Pull request merged**
